sn_networking/src/lib.rs

```diff
@@ -50,27 +24,57 @@ use libp2p::{
     Multiaddr, PeerId,
 };
 use rand::Rng;
+use tokio::sync::{
+    mpsc::{self, Sender},
+    oneshot,
+};
+use tokio::time::Duration;
+use tracing::trace;
+
 use sn_protocol::{
     error::Error as ProtocolError,
     messages::{ChunkProof, Nonce, Query, QueryResponse, Request, Response},
     storage::{RecordType, RetryStrategy},
     NetworkAddress, PrettyPrintKBucketKey, PrettyPrintRecordKey,
 };
 use sn_transfers::{MainPubkey, NanoTokens, PaymentQuote};
-use std::{
-    collections::{BTreeMap, HashMap},
-    path::PathBuf,
-};
-use tokio::sync::{
-    mpsc::{self, Sender},
-    oneshot,
+// re-export arch dependent deps for use in the crate, or above
+pub use target_arch::{interval, sleep, spawn, Instant, Interval};
+
+use self::{cmd::SwarmCmd, error::Result};
+pub use self::{
+    cmd::SwarmLocalState,
+    driver::{GetRecordCfg, NetworkBuilder, PutRecordCfg, SwarmDriver, VerificationKind},
+    error::{Error, GetRecordError},
+    event::{MsgResponder, NetworkEvent},
+    record_store::NodeRecordStore,
+    transfers::get_singed_spends_from_record,
 };
 
-use tokio::time::Duration;
-use tracing::trace;
+mod bootstrap;
```

Comment:
why the move down here? 

At least in house (not sure on rusty) convention is to have local import modules at the top. I'm not sure this move is necessary?